dask/dataframe/core.py

```diff
@@ -810,6 +810,34 @@ def min(self, axis=None, skipna=True):
                                  skipna=skipna, axis=axis,
                                  meta=meta)
 
+    def idxmaxmin(self, fn, axis=None, skipna=True):
+        if fn == 'idxmax':
+            idxfn = _idxmax
+        elif fn == 'idxmin':
+            idxfn = _idxmin
+        else:
+            raise ValueError('Function must be idxmax or idxmin')
+
+        axis = self._validate_axis(axis)
+        # if axis == 1:
+            # return map_partitions(idxfn, None, self,
+                                  # token=self._token_prefix + fn,
+                                  # skipna=skipna, axis=axis)
+        # else:
+        meta = getattr(self._meta_nonempty, fn)(axis=axis)
+        print(meta)
+        return aca([self], chunk=idxmaxmin_chunk, aggregate=idxmaxmin_agg,
+                    meta=meta, token=self._token_prefix + fn,
+                    known_divisions=self.known_divisions, fn=fn, axis=axis)
+
+    @derived_from(pd.DataFrame)
+    def idxmax(self, axis=None, skipna=True):
+        return self.idxmaxmin('idxmax', axis=axis, skipna=skipna)
+
+    @derived_from(pd.DataFrame)
+    def idxmin(self, axis=None, skipna=True):
+        return self.idxmaxmin('idxmin', axis=axis, skipna=skipna)
```

Comment:
My guess is that we could get rid of `idxmaxmin` and just repeat the `aca` call twice, once in each of `idxmax` and `idxmin`.  This would repeat the aca call but get rid of the extra function indirection and get rid of all of the checking around `fn`.